thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-comparison-range/component.js

```diff
@@ -0,0 +1,70 @@
+import Ember from 'ember';
+import moment from 'moment';
+import fetch from 'fetch';
+import { toFilters, toFilterMap, filterPrefix, toCurrentUrn, toBaselineUrn, appendFilters } from 'thirdeye-frontend/helpers/utils';
+import _ from 'lodash'
+
+/**
+ * Date format the date picker component expects
+ * @type String
+ *
+ */
+const serverDateFormat = 'YYYY-MM-DD HH:mm';
+
+export default Ember.Component.extend({
+  range: null, // [0, 0]
+
+  compareMode: null, // ""
+
+  onChange: null, // func (start, end, compareMode)
+
+  rangeOptions: {
+    'Last hour': [moment().subtract(1, 'hours').startOf('hour'), moment().startOf('hour')],
+    'Last 3 hours': [moment().subtract(3, 'hours').startOf('hour'), moment().startOf('hour')],
+    'Last 6 hours': [moment().subtract(6, 'hours').startOf('hour'), moment().startOf('hour')],
+    'Last 24 hours': [moment().subtract(24, 'hours').startOf('hour'), moment().startOf('hour')]
+  },
+
+  compareModeOptions: [
+    'WoW',
+    'Wo2W',
+    'Wo3W',
+    'Wo4W'
+  ],
+
+  maxDateFormatted: Ember.computed({
+    get() {
+      return moment().endOf('day').format(serverDateFormat);
+    }
+  }),
+
+  startFormatted: Ember.computed('start', {
+    get() {
```

Comment:
As per our in person conversation, if you can, make a short note of why you're having a `get()` here, referencing this documentation: https://guides.emberjs.com/v2.7.0/object-model/computed-properties/#toc_setting-computed-properties

Reply:
fixed